File: truss-compiler-maven-plugin/src/main/java/truss/compiler/generator/bound/xml/BoundSchema.java
package truss.compiler.generator.bound.xml;

import javax.xml.bind.annotation.*;
import java.util.ArrayList;
import java.util.List;

@XmlRootElement(name = "schema")
@XmlAccessorType(XmlAccessType.FIELD)
public class BoundSchema {
    @XmlAttribute
    private String packageName;
    @XmlElement(name = "class")
    private List<BoundClass> classes = new ArrayList<>();

    public String getPackageName() {
        return packageName;
    }

    public List<BoundClass> getClasses() {
        return classes;
    }
}

File: Truss.Compiler.Generate/Bound/Xml/BoundSchema.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace Truss.Compiler.Generate.Bound.Xml {
    [XmlRoot("schema")]
    public class BoundSchema {
        [XmlAttribute("packageName")]
        public string PackageName { get; private set; }

        [XmlElement("class")]
        public List<BoundClass> Classes { get; private set; }
    }
}
